image/nodejs.sh

```diff
@@ -4,8 +4,13 @@ source /pd_build/buildconfig
 set -x
 
 echo "+ Enabling Node Source APT repo"
-curl -sL https://deb.nodesource.com/setup_18.x | bash -
-apt-get update
+NODE_MAJOR=18
+sudo apt-get update
+sudo apt-get install -y ca-certificates curl gnupg
```

Comment:
`ca-certiifaces` and `curl` are already installed. `gpg` executable appears to be already available via the `gpg` package. Works for me without those two lines. Maybe they can be removed to simplify the script?

Reply:
Removed in b37d17f